### ISSUE TITLE
an other translation problem

### ISSUE THREAD

**Issue by Fa10175** [JustCarmen/fancy_treeview#50]
English text : 
> He married 2 times. The first time he married xxx. The second time he married yyy.

translation by francy treeview in french :
> Il s'est marié 2 fois. La début fois, il épouse xxx . La deuxième fois, il épouse yyyyy.

an better traduction :
> Il s'est marié 2 fois. La _première_ fois, il épouse xxx . La deuxième fois, il épouse yyyyy.

**Comment by JustCarmen** [comment]
Hi you can change this translation yourself at:

https://translate.webtrees.net/projects/justcarmen/fancy-tree-view/fr/

Translations will be automatically send to github and will be added to the next release. If you want to use the updated language file immediately after you altered a translation you can download the compiled mo file from the translation server.

**Comment by Fa10175** [comment]
for issue 2
i download justcarmen-fancy-tree-view-fr.po.
i look for "début" i found nothing
when i look for first i found:
#. I18N: first marriage
#: fancy_treeview/app/FancyTreeviewClass.php:551
msgid "first"
msgstr "premier"
that is right.

i do the test after reload fancy_treeview-1.7.5.zip and also using after
github download.

for issue 1
in justcarmen-fancy-tree-view-fr.po i found :
#. I18N: %s is the age of death in years; %s is a number, e.g. at the age
of 40
#: fancy_treeview/app/FancyTreeviewClass.php:1107
#, php-format
msgctxt "age in years"
msgid "at the age of %s"
msgstr "à l’âge de %s ans"

when the function return the age and it's an number of year is correct
because number
but when is under a year the function should return a number of month or
day and month or day

if we remove "ans" it should be ok when the function return some month or
day but not if it is a year.

So i prefer do nothing as translation modification
i cannot test these problem in an other language because i don't know.

i hope that this help you

Thanks
 Fa10175

2016-11-13 16:51 GMT+01:00 JustCarmen <notifications@github.com>:

**Comment by JustCarmen** [comment]
>> #. I18N: first marriage
#: fancy_treeview/app/FancyTreeviewClass.php:551
msgid "first"
msgstr "premier"
that is right.

The ordinals (first, second, third etc) are only used in the context "the first time he/she married" etc.

Shouldn't "first" be translated as "première" in stead of "premier"? It is always feminine in this context isn't it?

>> when the function return the age and it's an number of year is correct
because number
but when is under a year the function should return a number of month or
day and month or day

There are two strings in the translation file. One that represent the age in years and one that represent the age in days/months.

#. I18N: %s is the age of death in days/months; %s is a string, e.g. at the age of 2 months
#: fancy_treeview/app/FancyTreeviewClass.php:1105
#, php-format
msgctxt "age in days/months"
msgid "at the age of %s"
msgstr "à l’âge de %s"

#. I18N: %s is the age of death in years; %s is a number, e.g. at the age of 40
#: fancy_treeview/app/FancyTreeviewClass.php:1107
#, php-format
msgctxt "age in years"
msgid "at the age of %s"
msgstr "à l’âge de %s ans"

In the first string %s contains the addition "days" or "months".
In the second string %s is a number.

So can you be more specific what you exactly want me to change here? an example might help.

**Comment by Fa10175** [comment]
for issue 2 : 

> Shouldn't "first" be translated as "première" in stead of "premier"? It is always feminine in this context isn't it?

yes , but we read _**début**_ instead of _**première**_
![debut](https://cloud.githubusercontent.com/assets/10681601/20296617/327e5262-ab0c-11e6-924e-af634867d4dc.JPG)

for issue 1 : 
the text in english : he dead  **at the age of 13 months.** 
the text in french : il meurt  **à l’âge de 13 mois ans.** 

the right text should be : il meurt  **à l’âge de 13 mois.** 
![jour-ans](https://cloud.githubusercontent.com/assets/10681601/20296381/cb498f7c-ab0a-11e6-861f-d555b2c9edeb.JPG)

**Comment by Fa10175** [comment]
I attach a .ged file with these 2 examples
[tree1.zip](https://github.com/JustCarmen/fancy_treeview/files/591296/tree1.zip)

**Comment by JustCarmen** [comment]
Without changing anything to the language files I got this output after importing your ged file:

> Jean dupond, fils de Louis dupond et Marie durand, naquit le 1 janvier 1900 et mort le 1 février 1900 à l’âge de 31 jours.

This one looks correct.

> Louis dupond. Il s'est marié 2 fois. La premier fois, il épouse Marie durand. La deuxième fois, il épouse julie Sol.

I don't see the word debut but premier. Premier should be première as discussed. I've updated the language file. Now the output is:

> Louis dupond. Il s'est marié 2 fois. La première fois, il épouse Marie durand. La deuxième fois, il épouse julie Sol.

Which is what you want.

I've done the translation change at the webtrees translation server, so it could be a while before it is visible here at github (can take 24 hours, sometimes more), but I've included the latest file here (po and mo)

[fr.zip](https://github.com/JustCarmen/fancy_treeview/files/593111/fr.zip)

I suspect you were using an older mo-file. After the last release several changes were made to the French translation. Although there are no real code changes since the last release, I will make a new release soon which will include the latest updates to the language files.
